### PR TITLE
Added a Calendar API sample for ASP.NET MVC5

### PR DESCRIPTION
**Pull request by taradinoc**
Added a Calendar API sample demonstrating ASP.NET MVC5 login and access token capture.

---

image

---

image

---

image

### CODE REVIEW

File: Calendar.ASP.NET.MVC5/App_Start/MyClaimTypes.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Calendar.ASP.NET.MVC5
+{
+    internal static class MyClaimTypes
+    {
+        public const string GoogleUserId = "GoogleUserId";
```
Comment (peleyal) [inline Calendar.ASP.NET.MVC5/App_Start/MyClaimTypes.cs:26]: Remind me why do we need this part?

File: Calendar.ASP.NET.MVC5/App_Start/MyRequestedScopes.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+using Google.Apis.Calendar.v3;
+
+namespace Calendar.ASP.NET.MVC5
+{
+    internal static class MyRequestedScopes
```
Comment (peleyal) [inline Calendar.ASP.NET.MVC5/App_Start/MyRequestedScopes.cs:26]: [optional]
And do we want to have one file that contains scopes, clientid and types?

File: Calendar.ASP.NET.MVC5/Models/CalendarEventGroup.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Google.Apis.Calendar.v3.Data;
```
Comment (peleyal) [inline Calendar.ASP.NET.MVC5/Models/CalendarEventGroup.cs:20]: Add space here (between line 18 and 19)

File: Calendar.ASP.NET.MVC5/App_Start/MyClaimTypes.cs (hunk 1 above)
Comment (taradinoc) [inline Calendar.ASP.NET.MVC5/App_Start/MyClaimTypes.cs:26]: This is a key we use to associate the user's Google user ID with their ASP.NET user account. The access/refresh tokens are kept in the API client's data store, indexed by Google user ID.

File: Calendar.ASP.NET.MVC5/App_Start/MyRequestedScopes.cs (hunk 2 above)
Comment (taradinoc) [inline Calendar.ASP.NET.MVC5/App_Start/MyRequestedScopes.cs:26]: Personally, I prefer keeping them in different classes (and one class per file).

**Pull request merged**